crates/ironrdp-connector/src/lib.rs

```diff
@@ -161,6 +163,7 @@ pub struct Config {
     pub dig_product_id: String,
     pub client_dir: String,
     pub platform: capability_sets::MajorPlatformType,
+    pub hardware_id: Uuid,
```

Comment:
question: Another question: what exactly is an hardware ID? Is it really an UUID? Should we wrap `[u32; 5]` or some variation of this into a new type instead? In UUIDs, some bits are used to encode format’s variant and version, which may not be appropriate if an hardware ID is defined as something else. The docs: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpele/95ada355-e25e-4601-83fa-a4b00392a352

Reply:
I moved it to `[u32; 4]` - `PlatformID` part we care about so it doesn't makes sense to leave it for the user to provide. The user should provide the 4 parts of the data identifying (somehow 🤷 ) the computer (we will use `UUID` of the node but you're right that it doesn't have to be this way)